wagtail/core/tests/test_streamfield.py

```diff
@@ -241,13 +260,23 @@ def test_can_append(self):
         self.assertEqual(fetched_body[1].value, "bar")
 
 
+class TestJSONStreamValueAccess(TestStreamValueAccess):
+    model = JSONStreamModel
+
+    @skip("JSONField-based StreamField does not support storing non-json content.")
+    def test_can_read_non_json_content(self):
+        pass
```

Comment:
This test is for supporting the use case described in the following code:
https://github.com/wagtail/wagtail/blob/d38c664c1f127b4e99958ab5ec35845dfd3ad44b/wagtail/core/fields.py#L110-L116

We cannot support such use case for `JSONField`-based `StreamField` because the database requires the data to be valid JSON.